config/initializers/devise.rb

```diff
@@ -239,6 +239,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :facebook, ENV["FACEBOOK_CLIENT_ID"], ENV["FACEBOOK_CLIENT_SECRET"]
```

Comment:
What's your approach for setting these environment variables? Codeship can take care of itself, but we should probably use a tool like Figaro to manage this in dev, as well as for manual testing/deployments.

Also, whose keys are you using?